pdr_backend/predictoor/predictoor_agent.py

```diff
@@ -347,6 +360,34 @@ def get_ohlcv_data(self):
         return mergedohlcv_df
 
     @enforce_types
+    def check_balances(self) -> bool:
+        min_OCEAN_bal = self.ppss.predictoor_ss.stake_amount.to_wei()
```

Comment:
As discussed in standup, add two new properties: `OCEAN()` and `ROSE()`. From that, you'll be able to simplify `check_balances()` further. Including: you won't need local variables for balances or tokens.